client/webserver/site/template-builder/main.go

```diff
@@ -0,0 +1,96 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"regexp"
+
+	"decred.org/dcrdex/client/webserver/locales"
+)
+
+var (
+	workingDirectory, _ = os.Getwd()
+	keyRegExp           = regexp.MustCompile(`\[\[\[([a-zA-Z0-9 _.-]+)\]\]\]`)
+)
+
+func main() {
```

Comment:
I'm going back and forth on this. I also maybe think that we should just move it to **client/cmd**.

Reply:
I think it's correct under client/webserver/site somewhere because there's no need to actually build a CLI binary executable, and it would only apply in this folder.  This makes the most sense to me as a build operation (not a cmd app) executed wither via `go run ./template-builder/main.go` or `go generate`